javascript/javascript2/week1/spiritAnimalOptions.js

```diff
@@ -0,0 +1,46 @@
+const spiritNames = [
+  "The Butterfly",
+  "The Spider",
+  "The Crow",
+  "The Owl",
+  "The Hummingbird",
+  "The Hawk",
+  "The Grasshopper",
+  "The Eagle",
+  "The Lion",
+  "The Tiger",
+];
+
+function spiritNamesGenerator(name) {
+  return `${name} - ${spiritNames[parseInt(Math.random() * 10)]}`;
+}
+
+function showAnimalName() {
+  let userName = document.getElementById("inputName").value;
+  if (userName !== "") {
+    sptName.innerHTML = spiritNamesGenerator(userName);
+  } else {
+    sptName.innerHTML = "please provide your name";
+  }
+}
+const input = document.getElementById("inputName");
+const btn = document.querySelector(".btn1");
+
+const sptName = document.querySelector("h3");
+const userOptionSelect = document.getElementById("userOptions")
+let userChoice = "";
+userOptionSelect.addEventListener('click', ()=> {
+  userChoice = document.querySelector("input[name = userOption]:checked").value;
+  sptName.innerHTML = "please provide your name";
+  input.innerText= "";
```

Comment:
nice idea to reset input for name with each option... 👍  I think I didnt do it

Reply:
cool